### ISSUE TITLE
Readme files have placeholders and not formatted correctly

### ISSUE THREAD

**Issue by l3ender** [sonatype-nexus-community/nexus-format-archetype#42]
I'm looking through the readme files in this repo (trying to get started for first time plugin dev) and I noticed the readme files have odd placeholders in them and also some ill-formatted markdown.

I suspect the files are dynamically updated with some build-time process and perhaps something broke during the update?

Here is an example:
https://github.com/sonatype-nexus-community/nexus-format-archetype/blob/a083dcc5a0d0d6efcfdf4e6b86d24f7a4b4e4611/src/main/resources/archetype-resources/README.md

And a few screenshots:
![image](https://user-images.githubusercontent.com/2243641/103181819-0d7e1780-486b-11eb-95dd-0959b22aca50.png)
![image](https://user-images.githubusercontent.com/2243641/103181829-271f5f00-486b-11eb-9b61-d3b8498e13e4.png)

**Comment by bhamail** [comment]
Hi @l3ender, Thanks for reaching out! In fact, these odd looking placeholders are intentional. I'll try to explain below.

This project contains the source code of a [Maven Archetype](https://maven.apache.org/guides/introduction/introduction-to-archetypes.html). The odd looking markdown you are seeing is designed to be run through the templating system, and is using placeholders that will be populated by that system.

To use this Maven Archetype, you need only follow the steps here: [How to create a format](https://github.com/sonatype-nexus-community/nexus-format-archetype#how-to-create-a-format).
After following those steps, you should have a new java project (the nexus format with whatever name you provided), and all the odd placeholders should have valid values, populated according your answers to the prompts when following the above steps.

The source file you are referencing in your bug report is the raw input. For an example of what the processed output file should look like, see this file in the Integration Test Reference tree: https://github.com/sonatype-nexus-community/nexus-format-archetype/blob/master/src/test/resources/projects/it1/reference/README.md. However, even that reference file may have some substitution going on to deal with dynamic values that change when the template generation occurs.

So, you shouldn't need to worry about these odd placeholders unless you are contributing changes back to the archetype project: https://github.com/sonatype-nexus-community/nexus-format-archetype#how-to-contribute-to-this-archetype

I've added a link in the project README.md to the maven archetype docs (c3cd768d37eeb0ed78115fbf1274efadc2150c84). If you can think of more doc changes that would help clarify things, please do create a PR. I realize much of what is going on here is clear as mud.

**Comment by l3ender** [comment]
Thanks for the response and detail. The readme made more sense after I generated a project, and I should have come back here after and updated this issue.

I don't remember how I ended up viewing the archetype readme files (as opposed to main repo readme): I think I was either linked to it directly from a different site (Nexus docs, Jira, stack overflow, or other). Or perhaps I was browsing the repo some before running the archetype and stumbled across it then.

Either way, it clears up my confusion and I'll close out the [non] issue. Thanks!
